src/main/scala/org/constellation/rollback/RollbackService.scala

```diff
@@ -83,6 +83,8 @@ class RollbackService[F[_]: Concurrent](
   private def validateAccountBalance(accountBalances: AccountBalances): Either[RollbackException, Unit] =
     accountBalances.count(_._2 < 0) match {
       case 0 => Right(())
-      case _ => Left(InvalidBalances)
+      case _ =>
+        println(accountBalances.filter(_._2 < 0).toString())
```

Comment:
please remove before merging or change it into a Slf4j logger

Reply:
Leftover from test :)